scout/constants/case_tags.py

```diff
@@ -3,8 +3,9 @@
 # keys are used in the load report cli, while key_name is saved in case database documents
 CUSTOM_CASE_REPORTS = {
     "delivery": {"key_name": "delivery_report", "format": "HTML", "pdf_export": True},
-    "cnv": {"key_name": "cnv_report", "format": "PDF", "pdf_export": True},
+    "cnv": {"key_name": "cnv_report", "format": "PDF", "pdf_export": False},
     "cov_qc": {"key_name": "coverage_qc_report", "format": "HTML", "pdf_export": True},
+    "exe_ver": {"key_name": "exe_ver", "format": "YAML", "pdf_export": True},
```

Comment:
The displayed report name could be more self explanatory here, look at the other report types:

<img width="188" alt="image" src="https://user-images.githubusercontent.com/28093618/203705616-88e4694b-4e2e-431c-bc8b-9e7ea25848cc.png">


```suggestion
    "exe_ver": {"key_name": "bioinfo_software", "format": "YAML", "pdf_export": True},
```

Reply:
Or pipeline software, ..?